docs/howtos.md

```diff
@@ -3,6 +3,60 @@
 This page is an effort to give short examples for common tasks and will be
 extended over time.
 
+## Visualize network predictions
+Users can inspect the prediction results using the visualizer. Run `python examples/vis_pred.py` to see an example.
+
+First, initialize a `Visualizer` and set up `LabelLUT` as label names to visualize. Here we would like to visualize points from `SemanticKITTI`. The labels can be obtained by `get_label_to_names()`
+```python
+    kitti_labels = SemanticKITTI.get_label_to_names()
```

Comment:
We should add the import statements so users know what they need to include.

Reply:
done